### ISSUE TITLE
Clicks being blocked by UI outside of elements

### ISSUE THREAD

**Issue by Davst** [4rtzel/poe-archnemesis-scanner#51]
Don't know if this is possible to resove. But it feels like there is an invisible box that blocks interactions with the game in the area of the listed items.

If this can be addressed, great.. maybe it is a limitation of how the items are drawn?

**Comment by 4rtzel** [comment]
Yes, there is an invisible box around the items. It's unfortunate drawback of the way we draw the recipes on screen. We need to react to the mouse events so we capture them all.

I think it should be possible to implement "visual" transparency and "mouse event" transparency for everything except the images, but I don't really know how to do it in tk.
